pkg/sdk/testint/tasks_gen_integration_test.go

```diff
@@ -192,6 +188,123 @@ func TestInt_Tasks(t *testing.T) {
 		assertTaskWithOptions(t, task, request.GetName(), "", "", "", "", false, "", &otherId)
 	})
 
+	t.Run("create dag of tasks", func(t *testing.T) {
+		rootName := random.String()
+		rootId := sdk.NewSchemaObjectIdentifier(testDb(t).Name, testSchema(t).Name, rootName)
+
+		request := sdk.NewCreateTaskRequest(rootId, sql).WithSchedule(sdk.String("10 MINUTE"))
+		root := createTaskWithRequest(t, request)
+
+		require.Equal(t, []sdk.SchemaObjectIdentifier{}, root.Predecessors)
+
+		t1Name := random.String()
+		t1Id := sdk.NewSchemaObjectIdentifier(testDb(t).Name, testSchema(t).Name, t1Name)
+
+		request = sdk.NewCreateTaskRequest(t1Id, sql).WithAfter([]sdk.SchemaObjectIdentifier{rootId})
+		t1 := createTaskWithRequest(t, request)
+
+		require.Equal(t, []sdk.SchemaObjectIdentifier{rootId}, t1.Predecessors)
```

Comment:
Contains is a one-way check, which allows the collection to contain more. In other places, I do it by also checking the size and using contains (because order matters). But for single-element collections `Equal` is easier.